descriptors/ModuleDescriptor-template.json

```
@@ -221,6 +221,22 @@
         }
       ]
     },
+    {
+      "id": "patron-notice-policy-storage",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/patron-notice-policy-storage/patron-notice-policies",
+          "permissionsRequired": ["circulation-storage.patron-notice-policies.item.create"]
```

This also needs updating to reflect the renamed permission
```suggestion
          "permissionsRequired": ["circulation-storage.patron-notice-policies.item.post"]
```

---

thanks, it makes sense

---

fixed